docs/portal/team-members.md

```diff
@@ -60,14 +70,23 @@ To re-send the invitation for a team member, complete the following steps:
 
 ## Edit Information for a Team Member
 
-Admin users can update any of the values: First Name, Last Name, Email, Job Function, or Role.
+Notes re role:
+- Users with the Admin role can update any of these values, for any team member in their participant organization.
+- Users with the Operations role have read-only access to the Team Members page, but cannot change the information.
```

Comment:
I think 'and' makes more sense than 'but' here because read-only is synonymous with not being able to change information